babel.config.js

```diff
@@ -2,4 +2,5 @@
 
 module.exports = {
   presets: ['@babel/preset-env', '@babel/flow'],
+  plugins: ["@babel/plugin-proposal-optional-chaining"],
```

Comment:
This makes me a little uncomfortable because TypeScript is slow to adopt "new" language features. Does it seem to be working out OK, or is that one of the reasons why we've disabled the validation in **.vscode/settings.json**?

Reply:
I missed using optional-chaining from Swift, but no need to get too far ahead of ourselves here... done